src/views/Login/Login.jsx

```diff
@@ -39,25 +40,13 @@ export class Login extends React.Component {
       return
     }
 
-    const token = getCookieValue('token')
+    // transfer the token from the cookie to the window
+    const token = Cookies.get('token')
     if (token) {
       deleteCookie('token')
       api.setToken(token)
       this.props.dispatch(handleLoginSuccess())
       this.props.history.push('/loading')
-      return
-    }
-
-    const err = getQueryValue(window.location.search, 'error')
-    if (err) {
-      switch (err) {
-      case 'token':
-        this.props.history.push('/token')
-        return
-      case 'access_denied':
-        this.props.history.push('/accessdenied')
-        return
-      }
```

Comment:
Follow-up to 06d39ce8d28bb412c7c6aba9c38c86215251102e, since the query parameter doesn't seem to be getting set anywhere. We should make sure that the "access denied" flow with SAML is checked in manual QA, though.

Reply:
Don't we still need some of the above logic to handle the "access denied" case? https://github.com/18F/e-QIP-prototype/blob/1f0f9a6d4af5aa424963415aad45b3f4c7e89cf0/api/http/saml.go#L133

Reply:
Ahhh, you are right - missed that, sorry! Walked back those changes.